File: decompiled/3.17.1/be/ehealth/businessconnector/therlink/domain/requests/HasTherapeuticLinkRequest.java
package be.ehealth.businessconnector.therlink.domain.requests;

import be.ehealth.businessconnector.therlink.domain.Author;
import be.ehealth.businessconnector.therlink.domain.TherapeuticLink;
import be.ehealth.businessconnector.therlink.domain.TherapeuticLinkRequestType;
import org.joda.time.DateTime;

public class HasTherapeuticLinkRequest extends TherapeuticLinkRequestType {
   private static final long serialVersionUID = 1L;

   public HasTherapeuticLinkRequest(DateTime date, String id, Author author, TherapeuticLink link) {
      super(id, date, author, link);
   }
}

File: src/main/kotlin/org/taktik/connector/business/therlink/domain/requests/HasTherapeuticLinkRequest.kt
package org.taktik.connector.business.therlink.domain.requests

import org.taktik.connector.business.therlink.domain.Author
import org.taktik.connector.business.therlink.domain.TherapeuticLink
import org.taktik.connector.business.therlink.domain.TherapeuticLinkRequestType
import org.joda.time.DateTime

class HasTherapeuticLinkRequest(date: DateTime, id: String, author: Author, link: TherapeuticLink) : TherapeuticLinkRequestType(id, date, author, link) {
    companion object {
        private val serialVersionUID = 1L
    }
}
